packages/cactus-plugin-persistence-fabric/src/main/sql/schema.sql

```diff
@@ -54,6 +76,9 @@ ALTER TABLE ONLY fabric.block
 CREATE UNIQUE INDEX block_hash_unique_idx ON fabric.block USING btree (hash);
 CREATE UNIQUE INDEX block_number_unique_idx ON fabric.block USING btree (number);
 
+DROP INDEX IF EXISTS fabric.block_hash_unique_idx;
+DROP INDEX IF EXISTS fabric.block_number_unique_idx;
```

Comment:
Sorry for nitpicking, but it's better not to create these indexes in a first place, rather then creating and removing them. Please remove these lines